python2/pow.py: Add pow.py, another solution to proof-of-work challenge

jpentland opened the pull request:
Usage: pow.py yourname domain pattern

e.g. pow.py example example.com c0ffee

Will result in:
Result: example.14254635@example.com
sha1sum: c0ffeee700f7493f5bf5e760c0b59478eab2fd02

---

My email: joe.pentland+3931879@gmail.com
sha1sum: c0ffee4726c25ff55bd5f26c11fca7d1b0e8fdd1

Commit:
python2/pow.py: Add pow.py, another solution to proof-of-work challenge

Usage: pow.py yourname domain pattern

e.g. pow.py example example.com c0ffee

Will result in:
Result: example.14254635@example.com
sha1sum: c0ffeee700f7493f5bf5e760c0b59478eab2fd02

readevalprint commented on python2/pow.py:
```diff
@@ -0,0 +1,22 @@
+		sys.stderr.write('Usage: %s yourname domain pattern\n' % sys.argv[0])
+		sys.exit(1)
+
+	email = "%s.%s@%s" % (sys.argv[1], "%d", sys.argv[2])
```
Hey @jpentland thanks for the pull request. The `.` in `"%s.%s@%s"` will generally make a different email account than the one you are using. Look in to the `+` (plus sign) with email addresses. https://support.google.com/mail/answer/12096?hl=en

Commit:
python2/pow.py: Use '+' as separator to allow for GMail disposable addresses

Output example:
./pow.py example example.com c0ffee

Result: example+1899634@example.com
sha1sum: c0ffee9c56ed151525700240ccb034ab34f27239

readevalprint commented:
+1 good work!
